src/strnom.rs

```diff
@@ -123,15 +112,15 @@ macro_rules! alt {
 
     ($i:expr, $subrule:ident!( $($args:tt)*) | $($rest:tt)*) => {
         match $subrule!($i, $($args)*) {
-            res @ $crate::strnom::IResult::Done(_, _) => res,
+            res @ Ok(_) => res,
```

Comment:
I suppose in this crate it's OK to assume that Ok and Err will be the real Ok and Err in scope (as we only have in-crate consumers?)